### PR TITLE
add feature layer update geometry sample

### PR DESCRIPTION
**Pull request by rchintapalli**
@doneill can you review this sample

### CODE REVIEW

File: feature-layer-update-geometry/src/main/res/menu/menu_main.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto"
+      xmlns:tools="http://schemas.android.com/tools"
+      tools:context="com.esri.arcgisruntime.samples.featurelayerupdategeometry.MainActivity">
+</menu>
```
Comment (doneill) [inline feature-layer-update-geometry/src/main/res/menu/menu_main.xml]: This resource appears unused

File: feature-layer-update-geometry/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<resources>
+    <!-- Default screen margins, per the Android Design guidelines. -->
+    <dimen name="activity_horizontal_margin">16dp</dimen>
+    <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="fab_margin">16dp</dimen>
```
Comment (doneill) [inline feature-layer-update-geometry/src/main/res/values/dimens.xml]: `fab_margin` is an unused resource

**Comment by doneill** [comment]
@rchintapalli Sample looks good, just a couple of comments to removed unused resources.

**Comment by doneill** [comment]
@rchintapalli Looks good, will merge.

**Comment by rchintapalli** [comment]
@doneill removed unused resources

**Pull request merged**